test/ir/inference/CMakeLists.txt

```diff
@@ -47,9 +47,11 @@ if(WIN32)
   list(REMOVE_ITEM TEST_TRT_CONVERTER
        "test_trt_convert_quantize_dequantize_linear")
   list(REMOVE_ITEM TEST_TRT_IR_PASSES "test_trt_explicit_quantization")
-  list(REMOVE_ITEM TEST_TRT_IR_PASSES "test_trt_explicit_quantization_resnet")
-  list(REMOVE_ITEM TEST_TRT_IR_PASSES
-       "test_trt_explicit_quantization_mobilenet")
+  if(WITH_MKLDNN)
+    list(REMOVE_ITEM TEST_TRT_IR_PASSES "test_trt_explicit_quantization_resnet")
+    list(REMOVE_ITEM TEST_TRT_IR_PASSES
+         "test_trt_explicit_quantization_mobilenet")
```

Comment:
why these two tests are under WITH_MKLDNN scope?

Reply:
In pr [#57160](https://github.com/PaddlePaddle/Paddle/pull/57160), it has a logic error in cmake, it remove the test case but still set_tests_properties later.